### DISCUSSION TITLE
No desktop icon

### DISCUSSION THREAD

**Discussion by throbscottle** [vlachoudis/bCNC#1906] [General]
It's just a small thing. Packaged version (Arch) no longer works so I did the pip install (after dire warnings from the OS) to a local directory in my ~/.local/bin/ which works ok but there is no desktop shortcut and no bCNC icon available when I make one so now I am sad. 
Please help!

**Comment by JipeB** [comment]
The problem comes from python versions higher than 3.10 which require installation in a virtual environment.
To start bCNC from an icon on the desktop, you have to create a launcher: right click on the desktop and select the create a launcher menu.
